src/tvu.rs

```diff
@@ -126,14 +128,20 @@ impl Tvu {
             blob_index,
             l_last_entry_id.clone(),
             to_leader_sender,
-            entry_stream,
             ledger_signal_sender,
             ledger_signal_receiver,
         );
 
+        let (entry_stream_stage, entry_stream_receiver) = EntryStreamStage::new(
```

Comment:
Can we just just even add this stage if `entry_stream.is_none()` at this point?

Reply:
We can, yes. I was thinking that would make dealing with the JoinHandle ugly, but I guess it's the same number of Options either way. What do you think?

Reply:
About the same but I like how this way we don't spin up an extra stage/thread that does nothing when entry stream is disabled